src/promptflow/tests/sdk_cli_azure_test/e2etests/test_telemetry.py

```diff
@@ -312,33 +312,37 @@ def check_inner_call(*args, **kwargs):
         assert first_sdk_calls[-1] is True
 
     def test_scrub_fields(self):
-        from promptflow import PFClient
+        from promptflow._sdk._telemetry.logging_handler import PromptFlowSDKExporter
```

Comment:
according to the live test: https://github.com/microsoft/promptflow/actions/runs/8178373873/job/22362202591
there are 4 breaking test here:

- test_custom_event
- test_inner_function_call
- test_different_request_id
- test_scrub_fields

I understand current change will only take effect for the last one? will we target to fix them all in this PR?

Reply:
I have fixed **test_scrub_fields** and **test_custom_event**, and **test_inner_function_call** and **test_different_request_id** can run successfully on my local machine. It may not seem like a telemetry error, or they have been automatically fixed.

Reply:
it's wired that those two tests can also passed in main branch with live mode, but it constantly failed in daily live test

@D-W- please help take a look on these two tests: **test_inner_function_call** and **test_different_request_id**, you can find the workflow run in the above comment.